composer.json

```diff
@@ -9,28 +9,28 @@
       "email": "phillip.shipley@gmail.com"
     }
   ],
-  "minimum-stability": "stable",
   "require": {
-    "php": ">=7.4",
+    "php": ">=8.1",
     "ext-gmp": "*",
     "ext-json": "*",
     "ext-memcached": "*",
-    "simplesamlphp/simplesamlphp": "~1.18.6",
+    "simplesamlphp/simplesamlphp": "^1.19.6",
     "simplesamlphp/composer-module-installer": "1.1.8",
-    "silinternational/simplesamlphp-module-expirychecker": "~3.0.5",
-    "silinternational/simplesamlphp-module-silauth": "^7.0.0",
-    "silinternational/simplesamlphp-module-mfa": "^5.0.0",
-    "silinternational/simplesamlphp-module-profilereview": "~2.0.3",
-    "silinternational/ssp-utilities": "^1.0.7",
-    "silinternational/simplesamlphp-module-material": "^7.5.0",
-    "silinternational/simplesamlphp-module-sildisco": "~3.1.0",
-    "silinternational/php-env": "^2.1.1"
+    "silinternational/simplesamlphp-module-expirychecker": "3.1.0",
+    "silinternational/simplesamlphp-module-silauth": "7.1.0",
+    "silinternational/simplesamlphp-module-mfa": "5.1.0",
+    "silinternational/simplesamlphp-module-profilereview": "2.1.0",
+    "silinternational/ssp-utilities": "^1.1.0",
+    "silinternational/simplesamlphp-module-material": "7.6.0",
+    "silinternational/simplesamlphp-module-sildisco": "3.3.1",
```

Comment:
Several of these are locked in at exact versions. Is it feasible to change any/all of them to allow for at least bugfix version updates (e.g. `~3.1.0`), if not new-feature version updates (e.g. `^3.1.0`)? (Composer version constraint docs are [here](https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators).)